### PR TITLE
VIH-9772 - added duplicate user check

### PR DESCRIPTION
**Pull request by marcogagliardi**
https://tools.hmcts.net/jira/browse/VIH-9772

- added duplicate user check
- added error message
image

### CODE REVIEW

File: AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/upload-work-hours/upload-work-hours.component.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { Component } from '@angular/core';
 import { FileType } from '../../common/model/file-type';
 import { WorkHoursFileProcessorService } from '../services/work-hours-file-processor.service';
+import { error } from '@angular/compiler-cli/src/transformers/util';
```
Comment (phillipjchambers) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/upload-work-hours/upload-work-hours.component.ts:4]: Is this needed?

**Review by will-craig — COMMENTED** [review]
Not too sure about these additions to the package-lock.json?

File: AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/upload-work-hours/upload-work-hours.component.ts (hunk 1 above)
Comment (marcogagliardi) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/upload-work-hours/upload-work-hours.component.ts:4]: no it is no needed. I have removed it

**Comment by marcogagliardi** [comment]
@will-craig I am not sure about the package-lock.json either. As I have done npm install it automatically updated it

**Review by will-craig — APPROVED** [review]

**Review by phillipjchambers — APPROVED** [review]

**Pull request merged**